File: BeeCrowd C++/Iniciante/1134.cpp
#include <iostream>
using namespace std;

void conferir_combustivel(int opcoes, int &alcool, int &gasolina, int &diesel) {
    if (opcoes == 1) {
        alcool += 1;
    } else if (opcoes == 2) {
        gasolina += 1;
    } else if (opcoes == 3) {
        diesel += 1;
    }
}

int main() {
    int opcoes;
    int alcool = 0;
    int gasolina = 0;
    int diesel = 0;

    while (true) {
        cin >> opcoes;

        if (opcoes == 4) {
            break;
        } else if (opcoes == 1 || opcoes == 2 || opcoes == 3) {
            conferir_combustivel(opcoes, alcool, gasolina, diesel);
        }
    }
  
    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << alcool << endl;
    cout << "Gasolina: " << gasolina << endl;
    cout << "Diesel: " << diesel << endl;

    return 0;
}

File: BeeCrowd Python/Iniciante/1134.py
def conferir_combustivel(opcoes):
    global alcool,gasolina,diesel
  
    if opcoes == 1:
        alcool += 1
    elif opcoes == 2:
        gasolina += 1
    elif (opcoes == 3):
        diesel += 1
      
def main():
    global alcool,gasolina,diesel
  
    alcool = 0
    gasolina = 0
    diesel = 0
  
    while True:
        opcoes = int(input())

        if (opcoes == 4):
          break
        elif opcoes == 1 or opcoes == 2 or opcoes == 3:
          conferir_combustivel(opcoes)
          
    print("MUITO OBRIGADO")
    print(f"Alcool: {alcool}")
    print(f"Gasolina: {gasolina}")
    print(f"Diesel: {diesel}")
    
main()
